src/tcgwars/logic/impl/gen1/BaseSetNG.groovy

```diff
@@ -1943,7 +1943,45 @@ public enum BaseSetNG implements LogicCardInfo {
           }
         }
       case PLUSPOWER:
-        return copy(Unleashed.PLUSPOWER_80, this);
+        // This print increases after W/R, do not use it as copy for DP-era prints that do it before W/R (and affects both actives, while this one only affects the defending Pokémon).
+        return basicTrainer (this) {
+          text "Attach PlusPower to your Active Pokémon. At the end of your turn, discard PlusPower. If this Pokémon’s attack does damage to the Defending Pokémon (after applying Weakness and Resistance), the attack does 10 more damage to the Defending Pokémon."
+          def eff
+          onPlay {reason->
+            def pcs = my.active
+            if (my.bench) {
+              pcs = my.all.select("Which Pokémon will you attach $thisCard to?")
+            }
+            pcs.cards.add(thisCard)
+            my.hand.remove(thisCard)
+
+            eff = delayed {
+              before APPLY_ATTACK_DAMAGES, {
+                if (ef.attacker == pcs) {
+                  bg.dm().each {
+                    if (it.to.active && it.to.owner != pcs.owner && it.dmg.value) {
+                      bc "PlusPower +10"
+                      it.dmg += hp(10)
+                    }
+                  }
+                }
+              }
+              after DISCARD, {
+                if(ef.card == thisCard){
+                  eff.unregister()
+                }
+              }
```

Comment:
Discard should inherently call RemoveFromPlay since it's pretty much just a MoveCard. Is this definitely necessary?